cves/CVE-2011-2729.yml

```diff
@@ -202,4 +202,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: The requirements for the wrapper were not clearly specified.  Had they been, proper unit tests could have been developed.
```

Comment:
I would elaborate on this a bit more - maybe talk about what kinds of unit tests would be needed or what else could have helped prevent this vulnerability.

Reply:
The requirements should have the wrapper inputs and outputs and their associated data types and ranges. The requirements should also include how error are handled. The unit tests could include that type safety was guaranteed. The unit test could inject a broad range of bad data and bad data types to ensure that there are no unexpected behaviors. Bad data could include bad data types, out of range values, pointers that point to other sections of memory or routines, encapsulated programs, and malware.